p2pd/main.go

```diff
@@ -71,12 +78,17 @@ func main() {
 		opts = append(opts,
 			libp2p.DefaultTransports,
 			libp2p.Transport(quic.NewTransport),
-			libp2p.ListenAddrStrings(
-				"/ip4/0.0.0.0/tcp/0",
-				"/ip4/0.0.0.0/udp/0/quic",
-				"/ip6/::1/tcp/0",
-				"/ip6/::1/udp/0/quic",
-			))
+		)
+
+		if *hostAddrs == "" {
+			opts = append(opts,
+				libp2p.ListenAddrStrings(
```

Comment:
Not sure I get this special casing for QUIC?

Reply:
It's a subtlety of the libp2p constructor: If we specify a transport explicitly, then there are no default listen addrs...

Reply:
Got it, thanks.